server/stream.go

```diff
@@ -2029,7 +2029,12 @@ func (mset *stream) deleteMsg(seq uint64) (bool, error) {
 	}
 	mset.mu.RUnlock()
 
-	return mset.store.RemoveMsg(seq)
+	y, err := mset.store.RemoveMsg(seq)
```

Comment:
@kozlovic @neilalexander is there an existing utility to map Store -> JS API errors?

Reply:
I would revert changes to stream.go. See my other comment.